llvm/test/Transforms/LoopInterchange/pr47259.ll

```diff
@@ -0,0 +1,55 @@
+; REQUIRES: asserts
+; RUN: opt < %s -passes=loop-interchange -cache-line-size=64 -verify-dom-info -verify-loop-info \
+; RUN:     -S -debug 2>&1 | FileCheck %s
```

Comment:
`-disable-output -debug-only=loop-interchange`? Also, is `-cache-line-size=64` required for the test?